### DISCUSSION TITLE
v0.27.5

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/UnblockNeteaseMusic#1365] [Announcements]
## TL; DR;
- Allow customize NCM cookies
- Fixed pyncmd provider

## Announcement
So, this might be the last release due to the lack of maintenance, consider migrating to Spotify, we use this too.
Of course, feel free to take over the maintainership if you are interested in it :-)

## Fixes
- fix(provider/pyncmd): migrate api changes
- feat(hook): fallback to local time if 'now' property does not exist

## Features
* feat(hook): Add customize Netease Cookie
* feat(request): update user-agent to recent version

## Chore
- Update dependencies to the latest

## New Contributors
- @577fkj made their first contribution in https://github.com/UnblockNeteaseMusic/server/pull/1342

**Full Changelog**: https://github.com/UnblockNeteaseMusic/server/compare/v0.27.4-patch.1...v0.27.5

<hr /><em>This discussion was created from the release <a href='https://github.com/UnblockNeteaseMusic/server/releases/tag/v0.27.5'>v0.27.5</a>.</em>
